bin/svg-sprite.js

```diff
@@ -220,13 +219,17 @@ if (typeof config.shape.transform === 'string') {
 }
 
 // Run through all sprite modes
+// TODO
+// eslint-disable-next-line unicorn/no-array-for-each
 ['css', 'view', 'defs', 'symbol', 'stack'].forEach(function(mode) {
```

Comment:
This should also moved to constants in future

Reply:
Yeah, this and the other TODO would be nice to switch to for of, but I couldn't make it work myself. Feel free to push to this branch.